util/src/main/scala/stryker4s/extensions/mutationtypes/LiteralMutationTypes.scala

```diff
@@ -9,3 +9,29 @@ case object True extends LiteralMutation[Lit.Boolean] {
 case object False extends LiteralMutation[Lit.Boolean] {
   override val tree: Lit.Boolean = Lit.Boolean(false)
 }
+
+case object EmptyString extends LiteralMutation[Lit.String] {
+  override val tree: Lit.String = Lit.String("")
+
+  override def unapply(arg: Lit.String): Option[Lit.String] =
+    super.unapply(arg).filterNot(ParentIsInterpolatedString(_))
+}
+
+case object StrykerWasHereString extends LiteralMutation[Lit.String] {
+  override val tree: Lit.String = Lit.String("Stryker was here!")
+}
+
+/** Not a mutation, just an extractor for pattern matching
+  */
+case object NonEmptyString {
+  def unapply(arg: Lit.String): Option[Lit.String] =
+    Some(arg).filter(_.value.nonEmpty).filterNot(ParentIsInterpolatedString(_))
+}
+
+object ParentIsInterpolatedString {
+  def apply(arg: Lit.String): Boolean = arg.parent match {
+    // Do not mutate interpolated strings
```

Comment:
It can cause compile errors, which I think we should try to avoid until #14 is implemented. For example, changin `val a: Stat = q"val b = 0"` to `val a: Stat = q""` causes a compile error. 

Now that I'm thinking about it again, we could of course also only mutate string interpolation, and not others, like Scalameta's quasi-quotes...

Reply:
Yea i would agree on that because that's also what the comment is stating ;)

Reply:
I've added string interpolation mutators